### PR TITLE
labels for glossary

### PR DESCRIPTION
**Pull request by rssag**

### CODE REVIEW

File: content/glossary/r.md [hunk 1]
```diff
@@ -7,19 +7,19 @@ section:
 ---
 
 
-### Real-time processing
+### Real-time processing {#realtime-processing}
```
Comment (BeateRixen) [inline content/glossary/r.md:10]: > real-time

File: content/glossary/m.md [hunk 2]
```diff
@@ -47,22 +47,22 @@ Microservices are deployed as Container images to {{< product-c8y-iot >}}, and f
...
-### MQTT
+### MQTT {#mqtt}
 
 {{< product-c8y-iot >}} supports MQTT, an OASIS standard messaging protocol, for device integration.
 
 For details see [Device management > Device integration > Device integration using MQTT](/device-integration/mqtt).
 
 
-### Multi-tenancy
+### Multi-tenancy {#multitenancy}
```
Comment (rssag) [inline content/glossary/m.md:65]:
```suggestion
### Multi-tenancy {#multi-tenancy}
```

[the remark this answers is not in the document]
Comment (rssag) [reply content/glossary/m.md:65]: added

File: content/glossary/r.md (hunk 1 above)
Comment (rssag) [inline content/glossary/r.md:10]:
```suggestion
### Real-time processing {#real-time-processing}
```

Comment (rssag) [reply content/glossary/r.md:10]: added

**Review by BeateRixen — APPROVED** [review]

**Pull request merged**